app/serializers/user_bookmark_base_serializer.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_relative 'post_item_excerpt'
+
+class UserBookmarkBaseSerializer < ApplicationSerializer
+  include PostItemExcerpt
+
+  attributes :id,
+             :created_at,
+             :updated_at,
+             :name,
+             :reminder_at,
+             :pinned,
+             :title,
+             :fancy_title,
+             :bookmarkable_id,
+             :bookmarkable_type,
+             :bookmarkable_user_username,
+             :bookmarkable_user_avatar_template,
+             :bookmarkable_user_name,
+             :bookmarkable_url
+
+  def title
+    object.name
+  end
+
+  def fancy_title
+    object.name
+  end
+
+  def cooked
+    "-"
+  end
+
+  def bookmarkable_user_username
+    bookmarkable_user.username
+  end
+
+  def bookmarkable_user_avatar_template
+    bookmarkable_user.avatar_template
+  end
+
+  def bookmarkable_user_name
+    bookmarkable_user.name
+  end
```

Comment:
Could these be replaced by a `has_one :user, serializer: BasicUserSerializer, embed: :objects` here?

Reply:
Perhaps, I will try it out, mainly added these three as a kind of stub for this serializer to fit the new version of the bookmark list template.

Reply:
Yes this worked well, thanks :+1: